R/gcloud-storage.R

```diff
@@ -115,7 +115,7 @@ gs_rsync <- function(source, destination,
 gs_data_dir <- function(url, local_dir = "gs", force_sync = FALSE, echo = TRUE) {
 
   # if we are running on cloudml then just return the url unmodified
-  if (config::is_active("cloudml")) {
```

Comment:
We advertise here that the `cloudml` config will be active when running on cloudml: https://tensorflow.rstudio.com/tools/cloudml/articles/storage.html#data-source-configuration

If it's not active, we may have another bug to address (i.e. not sure if the proposed fix here is correct as it still leaves the other issue untouched)